### PR TITLE
add script for installing 2020.09 pilot stack

### LINKED ISSUE #13: Filter LD_LIBRARY_PATH
We should set:
```
filter-env-vars=LD_LIBRARY_PATH
```
in our Easybuild configuration, to prevent (system) tools from picking up libraries from modules, which will result in nasty errors.

### LINKED ISSUE #14: Open MPI configuration
The Open MPI installation included in the 2020.08 pilot software stack is configured with `--without-verbs`, so it has no support for high-speed interconnects.

### PR DESCRIPTION
**Pull request by boegel**
fixes #13, #17

WIP because I'm still hoping to fix #14 too...

To use: copy both `EESSI-pilot-2020.09.sh eessi_software_subdir.py` in the same directory, and call `EESSI-pilot-2020.09.sh` after setting up the Gentoo Prefix environment with `startprefix`

c @terjekv, @bedroge

### CODE REVIEW

File: EESSI-pilot-2020.09.sh [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+export EASYBUILD_SYSROOT=${EPREFIX}
+
+export EASYBUILD_DEBUG=1
+export EASYBUILD_TRACE=1
+
+export EASYBUILD_RPATH=1
+export EASYBUILD_FILTER_ENV_VARS=LD_LIBRARY_PATH
+
+export EASYBUILD_FILTER_DEPS=Autoconf,Automake,Autotools,binutils,bzip2,gettext,libreadline,libtool,M4,ncurses,XZ,zlib
```
Comment (boegel) [inline EESSI-pilot-2020.09.sh:76]: More stuff we should consider adding to `--filter-deps`:

* `Bison`
* `cURL`
* `expat`
* `flex`
* `help2man`
* `pkg-config`

**Comment by bedroge** [comment]
Nice, this is very useful! I'm running/testing it on a broadwell node now.

File: EESSI-pilot-2020.09.sh (hunk 1 above)
Comment (bedroge) [reply EESSI-pilot-2020.09.sh]: Do we still need to keep/discuss this somewhere (in an issue)?

**Review by bedroge — APPROVED** [review]
Lgtm, and this worked very well for building the pilot software for different architectures. I only had a minor issue with some hardcoded paths to /tmp (ran out of space).

File: EESSI-pilot-2020.09.sh (hunk 1 above)
Comment (boegel) [reply EESSI-pilot-2020.09.sh]: Yes, we should have a discussion on what we (won't) include in this list for the next iteration (2020.10).

We've hit several issues by leveraging the `binutils`, `zlib`, `ncurses` provided by Prefix, for different reasons.
That's not a reason to not filter those, but it does show that relying on stuff from the compat layer could have an impact (changes required in easyblocks/easyconfigs to make it work).

Comment (boegel) [reply EESSI-pilot-2020.09.sh]: see #23

**Pull request merged**